app/src/main/java/com/hover/stax/home/AbstractSDKCaller.kt

```diff
@@ -121,15 +120,9 @@ abstract class AbstractSDKCaller : AbstractGoogleAuthActivity(), PushNotificatio
 
     private fun addRecipientInfo(hsb: HoverSession.Builder) {
         hsb.extra(HoverAction.ACCOUNT_KEY, transferViewModel.contact.value!!.accountNumber)
-                .extra(
-                        HoverAction.PHONE_KEY, PhoneHelper.getNumberFormatForInput(
```

Comment:
Don't do normalization here, this is what the HoverSession class is for which already doing normalization on the phone number